napari/_qt/menus/file_menu.py

```diff
@@ -131,6 +132,32 @@ def __init__(self, window: 'Window'):
         self._rebuild_samples_menu()
         self.update()
 
+    def _close_app(self):
+        message = QMessageBox(
+            QMessageBox.Icon.Warning,
+            "Close confirm",
+            "Confirm to close application (could confirm with 'Ctrl+Q')",
```

Comment:
do we need translations on these messages?

Minor language suggestion
```suggestion
            "Close application?",
            "Confirm to close application (or press 'Ctrl+Q')",
```